plasmapy/utils/decorators/checks.py

```diff
@@ -263,11 +263,7 @@ def _get_value_checks(
                     # during subclassing)
                     out_checks[param.name][v_name] = v_default
 
-        # Does `self.checks` indicate arguments not used by f?
```

Comment:
Put back the comment.  I think the code is more readable with the comment.

Reply:
Agreed — I overlooked this one when I was re-adding comments.